src/i18n/es-ES.js

```diff
@@ -22,6 +22,7 @@ export default {
         volume_slider: 'Control deslizante de volumen',
         rewind_10: 'Rebobinar 10 segundos',
         play: 'Haz clic para reproducir',
+        replay: 'Haga clic para reproducir',
```

Comment:
@JRogaishio Should this be "Haga clic para repetir" ?

Reply:
@mwarner No idea. @tcedge would know since he's a native speaker.

Reply:
hi @JRogaishio  let's change a couple of things on the tranlations 
`      
        volume_slider: 'Control  sonido',
        rewind_10: 'volver 10 segundos atras',
        play: 'Haz click para ver el video ',
        replay: 'haz click para volver ver el video',
`

Reply:
@bigteddo Fixed and pushed.